### PR TITLE
some world stuff

### PR DESCRIPTION
**Pull request by BluCobalt**

### CODE REVIEW

File: mappings/net/minecraft/world/World.mapping [hunk 1]
```diff
@@ -220,7 +222,7 @@ CLASS net/minecraft/class_99 net/minecraft/world/World
 	METHOD method_335 (Lnet/minecraft/class_1745;Z)V
 		ARG 1 entity
 		ARG 2 bl
-	METHOD method_336 (Lnet/minecraft/class_1963;ILnet/minecraft/class_1372;I)V
+	METHOD method_336 pushPlayerLevelEvent (Lnet/minecraft/class_1963;ILnet/minecraft/class_1372;I)V
```
Comment (DeadlyMC) [inline mappings/net/minecraft/world/World.mapping:225]: `syncWorldEvent` , checkout ClientWorld in 1.16

File: mappings/net/minecraft/world/World.mapping [hunk 2]
```diff
@@ -220,7 +222,7 @@ CLASS net/minecraft/class_99 net/minecraft/world/World
...
 		ARG 1 player
 		ARG 2 type
```
Comment (DeadlyMC) [inline mappings/net/minecraft/world/World.mapping:227]: `eventId`

File: mappings/net/minecraft/world/World.mapping [hunk 3]
```diff
@@ -252,8 +254,10 @@ CLASS net/minecraft/class_99 net/minecraft/world/World
 		ARG 4 x2
 		ARG 5 y2
 		ARG 6 z2
-	METHOD method_354 (ILnet/minecraft/class_1372;I)V
+	METHOD method_354 pushLevelEvent (ILnet/minecraft/class_1372;I)V
```
Comment (DeadlyMC) [inline mappings/net/minecraft/world/World.mapping:257]: `syncGlobalEvent`, checkout ClientWorld

File: mappings/net/minecraft/world/World.mapping [hunk 4]
```diff
@@ -252,8 +254,10 @@ CLASS net/minecraft/class_99 net/minecraft/world/World
...
+		ARG 1 type
```
Comment (DeadlyMC) [inline mappings/net/minecraft/world/World.mapping:258]: eventId

File: mappings/net/minecraft/world/WorldEventListener.mapping [hunk 5]
```diff
@@ -30,6 +35,10 @@ CLASS net/minecraft/class_101 net/minecraft/world/WorldEventListener
 		ARG 7 z
 		ARG 9 volume
 		ARG 10 pitch
+	METHOD method_454 onBlockBreakInfo (ILnet/minecraft/class_1372;I)V
```
Comment (DeadlyMC) [inline mappings/net/minecraft/world/WorldEventListener.mapping:38]: `setBlockBreakingInfo`, upstream

**Review by DeadlyMC — CHANGES_REQUESTED** [review]
Added single comments for the requested changes

**Review by DeadlyMC — APPROVED** [review]

**Comment by BluCobalt** [comment]
@hYdos can you review this

**Review by hYdos — APPROVED** [review]

**Pull request merged**